.gitattributes

```diff
@@ -4,3 +4,10 @@
 .appveyor/    export-ignore
 .appveyor.yml export-ignore
 README        export-subst
+
+#Which files need CRLF handling
+* text=auto
+*.pl text eol=lf
+*.sh text eol=lf
+*.bat text eol=crlf
+*.cmd -text
```

Comment:
What is the motivation for treating `*.cmd` as binary?  Should this be `eol=lf`?

Reply:
There are unfortunatly 2 types of .cmd files:
a) Windows-ish (like .bat)
b) EPICS-ish (like st.cmd)
Don't mess with the line endings here, just to be sure.

Reply:
Also, if `.pl` need special handling, what about `.pm` and `.plt`, or `.pod`?

I understand that `.bat` must be crlf since the windows batch file parser is quite picky.  What about the others?  It would be good to document the motivations.

Reply:
```sh
$ git ls-files|grep 'cmd$'
modules/libcom/test/iocshTestBadArg.cmd
modules/libcom/test/iocshTestBadArgIndirect.cmd
modules/libcom/test/iocshTestSuccess.cmd
modules/libcom/test/iocshTestSuccessIndirect.cmd
```

We currently have only iocsh scripts with `.cmd`.

Reply:
OK @mdavidsaver @anjohnson : Thanks for the reviews.
I'll send a new patch tomorrow.